### PR TITLE
Add pin override for NUCLEO_F429ZI.

### PR DESCRIPTION
**Pull request by simosillankorva**
This is needed as the regular configuration does not work with this board.

### CODE REVIEW

**Comment by geky** [comment]
A quick question, is this only needed when ethernet is in use? It seemed to work fine in the default pinout otherwise.

File: README.md [hunk 1]
```diff
@@ -523,6 +523,16 @@ The above figure shows the K64F with the v1.0.0 CI test shield fitted. Note:
...
   
+### Wiring instructions for target NUCLEO_F429ZI with CI Test Shield
+
+![alt text](docs/pics/NUCLEO_F429ZI_wiring_with_ci_test_shield.png "unseen title text")
+
+**Figure 3. The figure shows how to connect the NUCLEO_F429ZI platform with the CI shield.**
+
+The above figure shows how to connect the NUCLEO_F429ZI with the v1.0.0 CI test shield. Note:
+
+- To get the SD Card to work with this platform the CI test shield cannot be connected directly to this board, instead follow the instruction above.
```
Comment (geky) [inline README.md]: Quick nit, could we add that this is only if ethernet is in use?

maybe
``` diff
- To get the SD Card to work with this platform the CI test shield cannot be connected directly to this board,
+ To get both Ethernet and the SD Card to work with this platform the CI test shield cannot be connected directly to this board,
```

Comment (JanneKiiskila) [inline README.md]: @VeliMattiLahtela commented there is a pin conflict also with Wi-Fi, can you please confirm?

**Comment by MarceloSalazar** [comment]
The image looks good, however the CI test shield is not publicly available nor there is requirement to use it.
Any SD card reader would work, as long as these lines are available through the SPI bus: SCLK, MISO, MOSI, CS. This is the key info to add to the docs.

File: README.md (hunk 1 above)
Comment (geky) [inline README.md]: I can confirm that at least the esp8266 does not have a pin conflict.

**Comment by geky** [comment]
Hey @JanneKiiskila, @Timjun01, this is low priority, but we're confused why this workaround is needed.

We followed the pins on the CI shield, but it looks like nothing conflicts with the NUCLEO board if you're only using the SD card. Is there a conflict with another peripheral?

cc @kegilbert

**Pull request merged**